data/virt_autotest/guest_params_xml_files/sles_16_64_kvm_hvm_sev_es_x86_64_agama_online_iso.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0"?>
+<guest>
+    <guest_os_name>sles</guest_os_name>
+    <guest_version>16.0</guest_version>
+    <guest_version_major>16</guest_version_major>
+    <guest_version_minor>0</guest_version_minor>
+    <guest_os_word_length>64</guest_os_word_length>
+    <guest_build/>
+    <host_hypervisor_uri/>
+    <host_virt_type>kvm</host_virt_type>
+    <guest_virt_type>hvm</guest_virt_type>
+    <guest_machine_type>q35</guest_machine_type>
+    <guest_arch>x86_64</guest_arch>
+    <guest_name/>
+    <guest_domain_name/>
+    <guest_memory>2048,maxmemory=4096</guest_memory>
+    <guest_vcpus>2,maxvcpus=4</guest_vcpus>
+    <guest_cpumodel>host-model</guest_cpumodel>
+    <guest_metadata/>
+    <guest_boot_settings>loader=/usr/share/qemu/ovmf-x86_64-sev.bin,hd,bootmenu.enable=yes,menu=on,loader.readonly=yes,loader.type=pflash,,loader.secure=no,loader.stateless=yes</guest_boot_settings>
```

Comment:
Hi Julie,
,,loader.secure=no ? Still not enabled, although firmware used?

Reply:
I saw it is "no" for all sev-es guests from sle15sp4 to sp7. I don't quite know what it means. @waynechen55 Should it be "yes"?

I am going to run a test with "yes".

Reply:
```
Now the 'secure' attribute has a bit misleading documentations as it doesn't control
whether the feature is enabled/disabled in the firmware but it is used to tell to
QEMU if the provided firmware is with secure boot feature enabled/disabled so QEMU
knows how to handle the firmware and access to it. We definitely need to improve
the documentation.
```

Reply:
My test failed with `loader.secure=yes`, https://openqa.suse.de/tests/17749946#step/unified_guest_installation/460

I think it shoud be "no" at here for sle16.

Reply:
From the failure screenshot, it shows secure boot needs SMM feature enabled. Anyway, sorry I messed this up with "uefi and secure boot". It is fine to ignore secure boot here.